### ISSUE TITLE
Source files are not part of the project

### ISSUE THREAD

**Issue by madhackademie** [Bigfoot71/Raymob#10]
Hi i have some trouble when i create header and cpp. Mostly i got this line on all my .cpp file : https://i.imgur.com/Lwfws3b.png
I dont know if i need to integrate this file in some Cmake list cuz look like include is not enought. If yes could you tell me how to do this ? Or do i miss something ?

**Comment by Bigfoot71** [comment]
@madhackademie This is due to the fact that Raymob's main CMakeLists.txt automatically includes all C/C++ files in your project for a quick setup when configuring the project with CMake (before building, see [L19](https://github.com/Bigfoot71/Raymob/blob/master/app/src/main/cpp/CMakeLists.txt#L19)).

Try the suggested synchronization, and if that's not sufficient, simply building your project will allow the IDE to integrate them during CMake configuration.

_You can also launch the configuration of CMake alone, without having to build the entire project, from the Build menu._

Alternatively, you can manually edit the main CMakeLists.txt to include your sources or use your own CMakeLists hierharchy for better organization, especially for larger projects.

By the way, you shouldn't need to specify the complete directory `deps/ray***` for inclusions, even if the IDE is finicky and temporarily claims the file is not integrated into the project.

**In summary, synchronize or build the project, or edit your own CMakeLists as needed.**

**Comment by Bigfoot71** [comment]
the problem surely comes from the fact that the file paths are not written "hard" in the CMakeLists, but when CMake searches for the C/C++ files the IDE should integrate them at the same time

**Comment by madhackademie** [comment]
Hi and bonne année. 
I have always the same error and i dont know if it's related but there is the behavior. whe i create a new class with .h and .cpp. There is an error like on the screenshot below 
https://i.imgur.com/Ce1WIRW.png
i will give you the code as well from the header and cpp (the main is the same as yours only my try to instanciate a bullet)
.h
```c++
#ifndef RAYMOB_BULLET_H
#define RAYMOB_BULLET_H

#include "deps/raylib/raylib.h"

class bullet {
public:
    bullet();
    explicit bullet(Vector2 pos, Vector2 velocity);

public:

private:
    Vector2 pos{};
    Vector2 velocity{};

};
#endif //RAYMOB_BULLET_H```

**Comment by madhackademie** [comment]
Hi sry for the late reply but i dont have the folder anymore a friend helped me and i do have a raymob project who is working so far. I do have a question related to folder in my project there are some red folder like in this screen shot 
![image](https://github.com/Bigfoot71/Raymob/assets/24621234/68668f53-774d-4871-bb00-5b48a9068248)

does this color mean something ?

**Comment by Bigfoot71** [comment]
Glad it's working for you!

Otherwise, the orange folders you're referring to are automatically generated by Android Studio, so there's no need to worry.
I'm not sure if it's to explicitly state that they are generated automatically or to mention that they are excluded from the Git repository, but in both cases, you can ignore them; it's normal.
